### DISCUSSION TITLE
Integrate pull request preview environments

### DISCUSSION THREAD

**Discussion by waveywaves** [AntlerVC/firetable#1147] [Idea / Feature Request]
**Is your feature request related to a problem? Please describe.**
I would like to support Rowy by implementing [Uffizzi](https://github.com/UffizziCloud/uffizzi) preview environments.
Disclaimer: I work on [Uffizzi](https://github.com/UffizziCloud/uffizzi).

Uffizzi is a Open Source full stack previews engine and our platform is available completely free for Rowy (and all open source projects). This will provide maintainers with preview environments of every PR in the cloud, which enables faster iterations and reduces time to merge. You can see the open source repos which are currently using Uffizzi over [here](https://uffizzi.notion.site/)

Uffizzi is purpose-built for the task of previewing PRs and it integrates with your workflow to deploy preview environments in the background without any manual steps for maintainers or contributors.

We can go ahead and create an Initial PoC for you right away if you think there is value in this proposal.

**Comment by shamsmosowi** [comment]
Hi @waveywaves,
Thanks for the offer. How does it differ from the vercel bot we have set up atm?

**Comment by waveywaves** [comment]
Hi @shamsmosowi, The backend that rowy will use on every PR preview will be isolated to the PR. The database can also contain seeded data if you would like that, in the meantime I will create a PoC for you to demonstrate this.
